lib/frizz/remote.rb

```diff
@@ -39,7 +50,9 @@ def delete(remote_file)
     attr_reader :bucket_name, :ignorance, :options
 
     def ignore?(object)
-      ignorance.ignore?(object.key)
+      # for remote files, you don't need to ignore for gzip preference,
+      # since there won't be any gzip files on s3
+      ignorance.ignore_matched_pattern?(object.key)
```

Comment:
Ok just made it through and the above is still interesting to me. I'd like to explore reducing the surface area of knowledge of this gzip feature... ideally everything shouldn't need to care about "gzip" in particular